### ISSUE TITLE
DC offset removal

### ISSUE THREAD

**Issue by MartinHerren** [demantz/RFAnalyzer#28]
A nice addition would be to have the possibility to remove the DC offset, to remove the peak on the tuning frequency.

**Comment by demantz** [comment]
Good point! As soon as I got the time ;) But it's now on my todo list..

**Comment by wissanu020430** [comment]
https://github.com/demantz/RFAnalyzer/issues/69#issuecomment-499084698

**Comment by BrunoKlu** [comment]
Yeees Would be great !! Thanks a lot
